dist/maptable.js

```diff
@@ -1021,7 +1172,7 @@ this.d3.maptable = (function () {
     }, {
       key: 'buildRow',
       value: function buildRow(filterName) {
-        var _this2 = this;
+        var _this3 = this;
```

Comment:
ack!  so true.  i didn't see i'd wondered off into the `dist/` directory - sorry!